sdk/go/pulumi/context_test.go

```diff
@@ -625,3 +626,36 @@ func TestWithValue(t *testing.T) {
 	assert.Equal(t, val, newCtx.Value(key))
 	assert.Equal(t, newCtx.state, testCtx.state)
 }
+
+func TestInvokeOutput(t *testing.T) {
+	t.Parallel()
+
+	mocks := &testMonitor{
+		CallF: func(args MockCallArgs) (resource.PropertyMap, error) {
+			if args.Token == "test:invoke:fail" {
+				return nil, errors.New("invoke error")
+			}
+			return resource.PropertyMap{"result": resource.NewStringProperty("success!")}, nil
+		},
+	}
+
+	type invokeArgs struct {
+		Arg string
+	}
+
+	err := RunErr(func(ctx *Context) error {
+		outType := AnyOutput{}
+		output := ctx.InvokeOutput("test:invoke:success", &invokeArgs{"will fail"}, outType, InvokeOutputOptions{})
```

Comment:
will fail?

Reply:
Oops, the argument is not used but I seem to have swapped them around